common/src/main/java/com/wynntils/models/gear/GearInfoRegistry.java

```diff
@@ -135,7 +135,7 @@ public GearInfo deserialize(JsonElement jsonElement, Type jsonType, JsonDeserial
                 throw new RuntimeException("Invalid Wynncraft data: item has no gear type");
             }
 
-            GearTier tier = GearTier.fromString(json.get("tier").getAsString());
+            GearTier tier = GearTier.fromString(json.get("rarity").getAsString());
```

Comment:
Do this reflect a general change in terminology from Wynncraft's side, that we should follow up in our code ("tier" -> "rarity")?

Reply:
I think this is just a semi random rename. They did change to using "armour" consistently, though.